projects/libexif/exif_loader_fuzzer.cc

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+#include <stdint.h>
+#include <libexif/exif-loader.h>
+
+
+void content_func(ExifEntry *entry, void *user_data) {
+  char buf[10000];
+  exif_entry_get_value(entry, buf, sizeof(buf));
+}
+
+void data_func(ExifContent *content, void *user_data) {
+  exif_content_foreach_entry(content, content_func, NULL);
+}
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size) {
+  ExifLoader *loader = exif_loader_new();
+  ExifData *data;
+  if (!loader) {
+    return 0;
+  }
+  exif_loader_write(loader, (unsigned char *)Data, Size);
```

Comment:
I'd recommend using C++ style casts in C++ code, e.g. `reinterpret_cast<unsigned char*>` in this case